py3status/modules/online_status.py

```diff
@@ -50,17 +55,18 @@ def _connection_present(self):
                                    stdout=fnull, stderr=fnull) == 0
 
     def online_status(self):
-        response = {
-            'cached_until': self.py3.time_in(self.cache_timeout)
-        }
+        response = {'cached_until': self.py3.time_in(self.cache_timeout)}
```

Comment:
Fixed (?). I jump in and try python. Please let me know if that's the wanted behaviour. It looks like `net_rate.py:157` may wrong too if it's going to constantly check things too.